tobiasHe/assignments/slidingwindow/lc209/lc209.js

```diff
@@ -1,25 +1,40 @@
+//Given an array of positive integers arr and a positive integer s,
+//return the minimal length of a contiguous subarray [numsl, numsl+1, ..., numsr-1, numsr] of
+//which the sum is greater than or equal to target. If there is no such subarray, return 0 instead.
+
+//s = 7, arr = [2,3,1,2,4,3]
+
+//s = 7, arr[7,3,1]  output: 1
+
+//output: 2
+
 function smallest_subarray_with_given_sum(s, arr) {
-  // TODO: Write code here
+	// TODO: Write code here
+	//whenever we find our targetSum = s;
+	//update our length of the subarray. Math.min(length, minLength)
+
+	// 7 , [ 2, 1, 5, 2, 3, 2 ])
+
+	let startWindow = 0;
+
```

Comment:
this space between each variable is a bit excessive, think about the code in "blocks" we can talk about this further if needed just LMK!